src/index.ts

```diff
@@ -39,8 +39,9 @@ module.exports = (app: Probot) => {
       }
     }
 
-    // reading pull request owner info and check it with configuration
-    const ownerSatisfied = config.from_owner.length === 0 || config.from_owner.includes(pr.user.login)
+    // reading pull request owner info and check it with configuration (case insensitive)
+    const lowerLogin = pr.user.login.toLowerCase()
+    const ownerSatisfied = config.from_owner.length === 0 || config.from_owner.some((owner: string) => owner.toLowerCase() === lowerLogin)
```

Comment:
That's an interesting case, out of curiosity: when can it happen that we have different cases of the owner? Or do you want to prevent a situation when GitHub username has different case than the one in the config file?

Reply:
I'm trying to address the situation where the `yml` file contains a different case from what GitHub has. Since GitHub handles are case-insensitive, it feels like the `yml` format should similarly not enforce casing.